### PR TITLE
Feature/560 - leaderboard by member

### PR DESCRIPTION
**Pull request by RomanGalochkin**
Please, change the base branch on "master" after all previous PRs are merged. Have done it to show code changes.

### CODE REVIEW

File: packages/client/src/components/LeaderboardTable.re [hunk 1]
```diff
@@ -53,7 +53,7 @@ module Columns = {
       ~render=
         (r: record) =>
           switch (r.pointScore) {
-          | Some(pointScore) => pointScore |> string_of_float |> Utils.ste
+          | Some(pointScore) => pointScore |> Js.Float.toString |> Utils.ste
```
Comment (OAGr) [inline packages/client/src/components/LeaderboardTable.re]: why? I prefer the E. library, so this could be standardized.

Comment (RomanGalochkin) [reply packages/client/src/components/LeaderboardTable.re]: string_of_float is deprecated

Comment (OAGr) [reply packages/client/src/components/LeaderboardTable.re]: Ah, can you put the functionality in the E library then, so all of our library use is one place, especially for ``Float``? Like, ``E.Float`` or whatever it is.

Comment (RomanGalochkin) [reply packages/client/src/components/LeaderboardTable.re]: yes, thanks

File: packages/client/src/components/LeaderboardTable.re [hunk 2]
```diff
@@ -80,7 +80,21 @@ module Columns = {
         (r: record) =>
           switch (r.predictionCountTotal) {
           | Some(predictionCountTotal) =>
-            predictionCountTotal |> string_of_int |> Utils.ste
+            predictionCountTotal |> Js.Int.toString |> Utils.ste
+          | _ => "0" |> Utils.ste
```
Comment (OAGr) [inline packages/client/src/components/LeaderboardTable.re]: This doesn't seem right to me. If the predictionCountTotal is empty, we don't know if it's zero. I'd prefer just showing an empty string or a null react element.

File: packages/client/src/graphql/AgentChannelsGet.re [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+      ~fn=
+        Query.make(
+          ~channelId?,
+          ~minNumberOfPredictions?,
+          ~minNumberOfQuestionsScored?,
+        ),
+      (),
+    );
+  componentMaker(query, innerComponentFn);
+};
```
Comment (OAGr) [inline packages/client/src/graphql/AgentChannelsGet.re]: Just curious, how long does it take you to write these files? They're kinda long and hypothetically we may be able to do something else to not need them to be so long later on.

Comment (RomanGalochkin) [reply packages/client/src/graphql/AgentChannelsGet.re]: Mmm, it was fast, so there is no hard need to make scaffolding or similar at the moment.

Comment (OAGr) [reply packages/client/src/graphql/AgentChannelsGet.re]: Cool, good work

File: packages/client/src/components/LeaderboardTable.re (hunk 2 above)
Comment (RomanGalochkin) [reply packages/client/src/components/LeaderboardTable.re]: Fixed.

File: packages/client/src/components/LeaderboardTable.re (hunk 1 above)
Comment (RomanGalochkin) [reply packages/client/src/components/LeaderboardTable.re]: Fixed.

**Comment by RomanGalochkin** [comment]
Updated.

**Comment by OAGr** [comment]
https://github.com/foretold-app/foretold/pull/589

**Pull request merged**